tools/workspace-plugin/src/plugins/shared.ts

```diff
@@ -12,3 +13,7 @@ export function assertProjectExists(projectRoot: string, context: CreateNodesCon
 
   return false;
 }
+
+export interface TaskBuilderConfig {
+  pmc?: ReturnType<typeof getPackageManagerCommand>;
```

Comment:
keeping optional for now, don't wanna force passing this everywhere if not needed